Background Scripts/listUsersHavingSpecificRole/script.js

```diff
@@ -0,0 +1,14 @@
+function getUsersBasedOnrole(role){
+	var userGR = new GlideRecord('sys_user');
+	var roleGR = new GlideRecord('sys_user_has_role');
+	var itilUsers = [];
+	roleGR.addQuery('role.name', role);
+	roleGR.query();
+	
+	while (roleGR.next()) {
+		userGR.get(roleGR.user);
```

Comment:
Instead of making a second call to the database you could use getDisplayValue from the initial lookup of user roles.